parsel/selector.py

```diff
@@ -173,30 +175,64 @@ def re_first(
             return el
         return default
 
-    def getall(self) -> List[str]:
+    def getall(
+        self,
+        text: bool = False,
+        cleaner: Union[str, None, Cleaner] = "auto",
+        guess_punct_space: bool = True,
+        guess_layout: bool = True,
```

Comment:
Maybe we could make these new arguments keyword-only?

Reply:
Absolutely we should do it, thanks for the suggestion!